src/da65/main.c

```diff
@@ -360,6 +361,51 @@ static void OptVersion (const char* Opt attribute ((unused)),
 
 
 
+static unsigned HandleChangedLength(const OpcDesc* D, unsigned PC)
+/* Instructions that have flSizeChanges set may use a different size than what
+** the table says. This function adjusts the PC accordingly, so after this only
+** the size from the table needs to be added to make up for the correct value
+*/
+{
+    if (D->Flags & flSizeChanges) {
+        if (CPU == CPU_45GS02) {
+            if ((D->Handler == OH_Implicit_42_45GS02)) {
```

Comment:
```suggestion
            if (D->Handler == OH_Implicit_42_45GS02) {
```

Or do you insist on double braces even for a single comparison?

Reply:
Haha no, how did i even create this.... :)